cardano-tracer/src/Cardano/Tracer/Handlers/RTView/Update/EKG.hs

```diff
@@ -12,11 +12,12 @@ import qualified Data.Map.Strict as M
 import           Data.Text (intercalate, isPrefixOf)
 import           Graphics.UI.Threepenny.Core (UI, liftIO)
 
+import           Cardano.Logging (showT)
+
 import           Cardano.Tracer.Environment
 import           Cardano.Tracer.Handlers.Metrics.Utils
 import           Cardano.Tracer.Handlers.RTView.UI.Utils
 import           Cardano.Tracer.Types
-import           Cardano.Tracer.Utils
```

Comment:
Probably, because here `Cardano.Tracer.Utils` is removed bug `Cardano.Logging` is added.

Should it be still exported from `Cardano.Tracer.Utils`?